symbulate/distributions.py

```diff
@@ -178,10 +178,12 @@ class Geometric(Distribution):
 
     def __init__(self, p):
         
-        if 0 <= p <= 1:
+        if 0 < p < 1:
             self.p = p
+        elif p == 1:    
+            raise Exception("Implement check for p = 1")
```

Comment:
Exceptions are supposed to be for the user, not reminders to yourself to implement something. If this is a TODO for you, you should raise NotImplementedError and put a comment reminding yourself to implement this later.